### ISSUE TITLE
Unable to render text in edito when open Settings Editor

### ISSUE THREAD

**Issue by git2013vb** [godotengine/godot#71262]
### Godot version

4 beta 11

### System information

Debian 11

### Issue description

![DeepinScreenshot_select-area_20230112080136](https://user-images.githubusercontent.com/13235119/211999940-a26532bf-c268-4382-885b-717ba7e8b317.png)

It happen randomly. Never happen with previous godot 4 betas

### Steps to reproduce

Try to change gdscript theme

### Minimal reproduction project

NA

**Comment by clayjohn** [comment]
Which renderer are you using?

**Comment by git2013vb** [comment]
> Which renderer are you using?

I'm using Compatibility mode

**Comment by Zireael07** [comment]
Strongly suspect it's a duplicate of #68814 (all the reports I've seen are with GLES 3)

EDIT: ninja'd

**Comment by git2013vb** [comment]
> Strongly suspect it's a duplicate of #68814 (all the reports I've seen are with GLES 3)
> 
> EDIT: ninja'd

I think so too. btw I have double monitors too, but in my case both monitors have default setting as  DPI (no scaling)

**Comment by Calinou** [comment]
Which graphics card model and driver are you using? Are you still using Bumblebee?

**Comment by git2013vb** [comment]
> Which graphics card model and driver are you using? Are you still using Bumblebee?

Nope, I usually buy the latest computer each time I found an error in Godot, why?
You surprise me for once. Still remember my past and unresolved problems. That's good.
So I'm expect that this one land into so called:
"a non specified bug of nvidia driver for unsupported old version and so on that is not worth to fix because you are the only one that have it"? But, alas, it seems someone else may have same issue on this one. :)

For a long time I restrained myself to open new issue in Godot just because I have the habit to find the weirdest bugs ever :). 
But now I feel maybe I try to avoid the weirdest one and focus the "normal" ones.. But Seems I still have to restrict my subset again..;)

Lets see what will happen. :) Stay tuned in this channel...

**Comment by akien-mga** [comment]
@git2013vb Please tone it down on the use of irony and passive aggressiveness. I believe it's not the first time I ask that of you in regard to the topic of you hardware and software issues.

It is a fact that old Nvidia chipsets on old Linux distroa have poor drivers, and those drivers have bugs. Those bugs trigger issues in Godot. It's not your fault and it's not our fault either.

We do what we can to accommodate as much hardware and software configurations as we can be we dont have infinite time nor infinite access to all hardware in the world to write workarounds for each bug that Nvidia, AMD or Intel would have left in their bitrotting legacy drivers.

---

Now I'm not saying this is the case for this specific issue. Calinou just asked about it to have additional context which can be useful, that's all.

**Comment by git2013vb** [comment]
When Godot work and a contributor break it of course is not MY fault. We are clear about this. 

The because I'm the only user related to that "regression" I was told there is nobody interested/capable to fix it.

Now is not important if I use or not that feature. But if I were the one who did the damage I will not sleep at night until I fix what I broke. Because I have the max respect to other people. Most of the time I expect the same from everyone, But that's me, and I know the reality is different. I can understand some people give a different level of importance to other users.

People have to be responsible of their own actions. It is a moral rather than a material question. 
If they leave who is in charge/manage have to take action to reduce/fix the "regression"

This is what i do and this is what I expect from others. Zero tolerance from my side - unless real emergency ofc. Sorry this is how I live.

Now the other days I tried to look into the code base to try to debug it - not only for my issues ofc - but I realize that because of many macros was impossible to me to be effective from the beginning.

Do you have plans to convert all macros in inline functions? 
==============
To answer to @Calinou : Yes I have the same hardware. Not because I cannot buy a better one but because I hate consumerism and  money. That's me. I'm a rare one. 

Thanks

**Comment by YuriSizov** [comment]
@git2013vb I'm sorry, but your combination of hardware and hardware drivers is outdated and is supported on the best-effort bases. This means that it's going to receive fixes if those fixes can be done within reasonable amount of contributors time, or if they happen to fix issues with other hardware or drivers.

I would like to remind you that projects under the MIT license are provided "[AS IS](https://github.com/godotengine/godot/blob/master/LICENSE.txt)". This means that we are not obligated to fix any issue, or provide support to any user. Once again, we try our best to support the project for a vast variety of devices, but under no circumstances can we dedicate an inordinate amount of time to fix individual issues that happen on individual, very specific hardware and software combinations. You are free to hire developers to keep the project stable on your machine, and equally free to never contribute those fixes to the upstream repository if you so desire.

Your hostile attitude towards project maintainers is not appreciated. Especially towards maintainers who have spent enough time on your personally to remember and recall your specific hardware and software configuration. This is pretty much an unprecedented amount of attention and mental capacity for one specific user. I would like to direct you to our [code of conduct](https://godotengine.org/code-of-conduct), and remind you that if you continue to behave this way you will be barred from interacting with maintainers on this platform.

**Comment by git2013vb** [comment]
>Your hostile attitude towards project maintainers is not appreciated. 

Apologise for that. Honestly was not my intention at all. Sorry again. I was expressing my thoughts. 
But I can assure It will never happen again.

I know already about MIT,  Code of Conduit and AS IS, of course. 
Again, was not my intend to make uncomfortable anyone.
I'm the first one who try to help other if I can.

>You are free to hire developers to keep the project stable on your machine, and equally free to never contribute those fixes to the upstream repository if you so desire.
Thanks but I already know I'm free :)

Sorry again to everyone.

Thanks.
